.erb-lint.yml

```diff
@@ -0,0 +1,69 @@
+---
+glob: "**/*.{html,text,js}{+*,}.erb"
+exclude:
+  - '**/app/components/status_card/view.html.erb'
+EnableDefaultLinters: true
+linters:
+  Rubocop:
+    enabled: true
+    rubocop_config:
+      inherit_from:
+        - .rubocop.yml
+      Style/FrozenStringLiteralComment:
+        Enabled: false # 666 app errors
```

Comment:
These are all disabled as they are causing the linter to fail. I am planning to address them in a few smaller PR's. The new gem `erb_lint` has picked up all these small problems.

Reply:
Sounds good! We should probably disabled the ones the don't make much sense in `erb` files anyway. This will probably cover all the large failures.

The example config from the gem's documentation is:


```
linters:
  Rubocop:
    enabled: true
    rubocop_config:
      inherit_from:
        - .rubocop.yml
      Layout/InitialIndentation:
        Enabled: false
      Layout/LineLength:
        Enabled: false
      Layout/TrailingEmptyLines:
        Enabled: false
      Layout/TrailingWhitespace:
        Enabled: false
      Naming/FileName:
        Enabled: false
      Style/FrozenStringLiteralComment:
        Enabled: false
      Lint/UselessAssignment:
        Enabled: false
      Rails/OutputSafety:
        Enabled: false
```

All the other ones, we should probably see whether we can fix.

Reply:
https://trello.com/c/FJesgPPN/1268-fix-small-errors-that-the-erblinter-has-picked-up